hyperscale/006-other_extensions.sql

```diff
@@ -0,0 +1,25 @@
+CREATE EXTENSION hstore;
+CREATE EXTENSION plperlu;
+CREATE EXTENSION jsonb_plperlu;
```

Comment:
can remove plperlu and jsonb_plperlu, they are not created on Hyperscale

Reply:
Removed

Reply:
You're right I fixed them